python/vyos/firewall.py

```diff
@@ -496,6 +496,19 @@ def parse_rule(rule_conf, hook, fw_name, rule_id, ip_name):
             output.append(f'vlan id {rule_conf["vlan"]["id"]}')
         if 'priority' in rule_conf['vlan']:
             output.append(f'vlan pcp {rule_conf["vlan"]["priority"]}')
+        if 'ethernet_type' in rule_conf['vlan']:
```

Comment:
Can we use `ether-type` without VLAN?

Reply:
Yes, we can use both:
- Without vlan, added previously: https://github.com/vyos/vyos-1x/blob/current/python/vyos/firewall.py#L154-L166
- And now extending the feature in order to match if types when vlan header is present.